src/Akka.Persistence.Azure/Journal/AzureTableStorageJournal.cs

```diff
@@ -285,7 +285,7 @@ protected override bool ReceivePluginInternal(object message)
             {
                 case ReplayTaggedMessages replay:
                     ReplayTaggedMessagesAsync(replay, _shutdownCts.Token)
-                        .PipeTo(replay.ReplyTo, success: h => new RecoverySuccess(h), failure: e => new ReplayMessagesFailure(e));
```

Comment:
Ah, is this what was truncating the results stream? Pre-emptively declaring the mission complete?

Reply:
Sort of, its only half of the story